src/settings/OwnerSettings.js

```diff
@@ -43,6 +43,16 @@ class OwnerSettings extends React.Component {
     this.connectedControlledVocab = props.stripes.connect(ControlledVocab);
   }
 
+  validate = (item, index, items) => {
```

Comment:
Rather confusing to have a function and a method both named `validate`. Maybe call this one `validateItem`, or rename the import?

Reply:
I understand, we will verify.